src/meta/src/barrier/backfill_order_control.rs

```diff
@@ -204,4 +204,47 @@ impl BackfillOrderState {
     pub fn get_locality_fragment_state_table_mapping(&self) -> &HashMap<FragmentId, Vec<TableId>> {
         &self.locality_fragment_state_table_mapping
     }
+
+    /// Refresh actor mapping after reschedule and return newly scheduled fragments.
+    pub fn refresh_actors(
+        &mut self,
+        fragment_actors: &HashMap<FragmentId, HashSet<ActorId>>,
+        done_actors: &HashSet<ActorId>,
+    ) -> Vec<FragmentId> {
+        self.actor_to_fragment_id = fragment_actors
+            .iter()
+            .flat_map(|(fragment_id, actors)| {
+                actors.iter().map(|actor_id| (*actor_id, *fragment_id))
+            })
+            .collect();
+
+        for node in self
+            .current_backfill_nodes
+            .values_mut()
+            .chain(self.remaining_backfill_nodes.values_mut())
+        {
+            if let Some(actors) = fragment_actors.get(&node.fragment_id) {
+                node.remaining_actors = actors
+                    .iter()
+                    .filter(|actor_id| !done_actors.contains(actor_id))
```

Comment:
Can we just use old actor_ids directly like this? I'm not sure if the `actor_id` is stable after online scaling.

Let's say we have the following scenario:
```
--- start, parallelism: 2 ---
Actor with id: 0
Actor with id: 1

--- scale in, parallelism: 1 ---

Actor with id: 0 ------> even though it has the same id, I suppose the state will be different
```

Shall we just queue all actors?